.cspell.json

```diff
@@ -781,6 +782,7 @@
     "inpaints",
     "inplace",
     "insertafter",
+    "interpolable",
```

Comment:
There is an alternative form. How about replacing it?
https://ejje.weblio.jp/content/interpolatable

Reply:
```suggestion
    "interpolatable",
```